3rdparty/stdgpu/stdgpu.cmake

```diff
@@ -28,5 +29,5 @@ ExternalProject_Add(
 
 ExternalProject_Get_Property(ext_stdgpu INSTALL_DIR)
 set(STDGPU_INCLUDE_DIRS ${INSTALL_DIR}/include/) # "/" is critical.
-set(STDGPU_LIB_DIR ${INSTALL_DIR}/lib)
+set(STDGPU_LIB_DIR ${INSTALL_DIR}/${Open3D_INSTALL_LIB_DIR})
```

Comment:
Probably invalid: the project does not respect conventions (see [here](https://github.com/stotko/stdgpu/blob/e10f6f3ccc9902d693af4380c3bcd188ec34a2e6/CMakeLists.txt#L139)).